guide/oauth2/README.md

```diff
@@ -0,0 +1,200 @@
+# OAuth2 
+
+OAuth2 enables application developers to build applications that utilize authentication and data from the Discord API.
+
+## A quick example
+
+###  Setting up a basic web server
+
+Typically, OAuth2 is used in websites to get information about their users from an external service. Here, we will use Node.js' built-in `http` module to create a web server because our example is simple enough. First, we create a file named `index.js` which will be used to start the server.
+
+```js
+const http = require('http');
+const fs = require('fs');
+const port = 53134;
+
+http.createServer((req, res) => {
+	let responseCode = 404;
+	let content = '404 Error';
+	if (req.url === '/') {
+		responseCode = 200;
+		content = fs.readFileSync('./index.html');
+	}
+	res.writeHead(responseCode, {
+		'content-type': 'text/html;charset=utf-8',
+	});
+	res.write(content);
+	res.end();
+}).listen(port);
+```
+
+Right now, we've designated that the contents of an `index.html` file will be served to the user when they visit the root domain, so let's create an `index.html` file in the same directory with the following contents.
+
+```html
+<!DOCTYPE html>
+<div id='info'>
+    Hoi!
+</div>
```

Comment:
Needs some `<head>` and `<body>` tags and tag attributes should use double quotes. The `<head>` tag can contain just a `<title>`.